app/components/UI/Tokens/index.js

```diff
@@ -266,7 +266,7 @@ class Tokens extends PureComponent {
 		const tokensToDisplay = hideZeroBalanceTokens
 			? tokens.filter(token => {
 					const { address, symbol } = token;
-					return (tokenBalances[address] && !tokenBalances[address].isZero()) || symbol === 'ETH';
+					return (tokenBalances[address] && !tokenBalances[address]?.isZero()) || symbol === 'ETH';
```

Comment:
I had the same issue today but `isZero` was defined but not a method

```suggestion
					return (tokenBalances[address] && !tokenBalances[address]?.isZero?.()) || symbol === 'ETH';
```